canonical-web-and-design/ubuntu.com
# Implement /blog

## Done

- implemented 
  - blog article template
  - tag index template
  - paginated list of all author's posts
- updated blog links in navigation to point to u.c/blog

## QA

- check out this feature branch
- Run `./run`
- Visit [/blog](http://0.0.0.0:8001/blog)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Ensure that pages on /blog looks and behave similarly to pages on https://blog.ubuntu.com/


## Issue / Card
Fixes #5263
